src/ripple/nodestore/NodeObject.h

```diff
@@ -95,8 +95,8 @@ class NodeObject : public CountedObject <NodeObject>
     Blob const& getData () const;
 
 private:
-    NodeObjectType mType;
-    uint256 mHash;
+    NodeObjectType const mType;
+    uint256 const mHash;
     Blob mData;
```

Comment:
You got half of it.  The change to the constructor looks good.  But you forgot to mark `mData` as `const`.  Not a big deal.

Reply:
Bah! You're right, will fix.